xpk.py

```diff
@@ -3200,6 +3200,37 @@ def setup_docker_image(args) -> tuple[int, str]:
 
   return 0, docker_image
 
+def check_use_pathways(args) -> bool:
+  """ 
+  Both --proxy-server-image and --server-image need to be set if --use-pathways is set. 
+  Neither can be provided if --use-pathways is not provided.
+  Args:
+    args: user provided arguments for running the command.
+
+  Returns:
+    bool:
+      yaml for main and sidecar container
+  """
+  return args.use_pathways == (args.proxy_server_image is not None and args.server_image is not None)
+
+def validate_pathways_docker_images(args) -> bool:
+  """Validates the existence of Pathways server and proxy docker images in the project.
+
+  Args:
+      args (_type_): _description_
+
+  Returns:
+      bool: _description_
+  """
+  # pylint: disable=line-too-long
```

Comment:
nit I don't think the line is too long

Reply:
Thanks for catching that! It was long previously!